Makefile

```diff
@@ -16,8 +18,11 @@ mkimage:
 		.
 
 .PHONY: iso
-iso:
-	ALPINE_VERSION=$(ALPINE_VERSION) REPO_VERSION=$(REPO_VERSION) EDITION=$(EDITION) ./build.sh
+iso: nerdctl-$(NERDCTL_VERSION)
+	ALPINE_VERSION=$(ALPINE_VERSION) NERDCTL_VERSION=$(NERDCTL_VERSION) REPO_VERSION=$(REPO_VERSION) EDITION=$(EDITION) ./build.sh
+
+nerdctl-$(NERDCTL_VERSION):
+	curl -o $@ -Ls https://github.com/containerd/nerdctl/releases/download/v$(NERDCTL_VERSION)/nerdctl-full-$(NERDCTL_VERSION)-linux-amd64.tar.gz
```

Comment:
This package currently contains dynamically-linked binaries and does not work on Alpine.

`apk add libc6-compat` doesn't seem enough

```console
# ./containerd
Error relocating ./containerd: __fprintf_chk: symbol not found
Error relocating ./containerd: __vfprintf_chk: symbol not found
```

Reply:
Note that we will be using the `containerd` from `k3s`, so we really only going to run `buildctl`, `buildkitd`, and `nerdctl`. Nothing else even gets copied into the image.

Reply:
Turns out we also need the `libexec` subtree... 😄